lightning/src/ln/channelmanager.rs

```diff
@@ -2764,10 +2764,10 @@ where
 	/// Send a payment that is probing the given route for liquidity. We calculate the
 	/// [`PaymentHash`] of probes based on a static secret and a random [`PaymentId`], which allows
 	/// us to easily discern them from real payments.
-	pub fn send_probe(&self, hops: Vec<RouteHop>) -> Result<(PaymentHash, PaymentId), PaymentSendFailure> {
+	pub fn send_probe(&self, path: Path) -> Result<(PaymentHash, PaymentId), PaymentSendFailure> {
```

Comment:
Worth mentioning in the changelog?

Reply:
I added a changelog for the backwards compat, though I'm not sure if it's helpful. Usually we don't try too hard to include things in the changelog that the compiler will make obvious upon upgrade :)